tests/_utils/model.js

```diff
@@ -365,3 +379,35 @@ function convertToModelText( withAttributes = false ) {
 		evt.stop();
 	};
 }
+
+// Typecast string value to the original type.
+// 		`'true'` => `true`
+// 		`'1'` => `1`
+//		`'{"x":1,"y":2}'` => `{ x: 1, y: 2 }`
+function parseValue( attribute ) {
+	try {
+		return JSON.parse( attribute );
+	} catch ( e ) {
+		return attribute;
+	}
+}
+
+// When value is an Object stringify it.
+function stringifyObject( data ) {
+	if ( isPlainObject( data ) ) {
+		return JSON.stringify( data );
+	}
+
+	return data;
+}
+
+// Loop trough attributes map and parse each value by passed parser.
+function parseAttributes( attributes, parser ) {
```

Comment:
Maybe `convertAttributes` and `converter`. The fact that `parser` can be `stringify` is misleading.